packages/web3-eth-contract/src/index.js

```diff
@@ -903,6 +904,26 @@ Contract.prototype._executeMethod = function _executeMethod(){
     }
 
     switch (args.type) {
+        case 'createAccessList':
+
+            // return error, if no "from" is specified
+            if(!utils.isAddress(args.options.from)) {
+                return utils._fireError(errors.ContractNoFromAddressDefinedError(), defer.eventEmitter, defer.reject, args.callback);
+            }
+
+            var createAccessList = (new Method({
+                name: 'createAccessList',
+                call: 'eth_createAccessList',
+                params: 2,
```

Comment:
Geth needs two [params](https://github.com/ethereum/go-ethereum/pull/22550/files#diff-c426ecd2f7d247753b9ea8c1cc003f21fa412661c1f967d203d4edf8163da344R1332) blockNum and Tx but If you are following and also want to add support of [nethermind](https://docs.nethermind.io/nethermind/ethereum-client/json-rpc/eth#eth_createaccesslist) it have 3rd optional param `optimize Boolean`  as well

Reply:
@jdevcs How do we have two optional parameters though?

Reply:
May be web3 automatically filling in `blockParameter` 'latest' if `optimize` is provided and `blockParameter` not. But it will make this call incompatible with Geth, if some one is passing 3rd param? Lets ignore this 3rd param so it works with both nethermind and geth for now.